src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -55,11 +62,15 @@ class HaAutomationEditor extends LitElement {
       _errors: {},
       _dirty: {},
       _config: {},
+      _showYaml: {},
+      _yaml: {},
+      _yamlEditor: {},
```

Comment:
`_yamlEditor ` shouldn't be a property, you don't use it for rendering. Every time a property changes the render function is called.